test/unit/sql/enum.test.js

```diff
@@ -43,13 +43,13 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
       describe('pgEnum', () => {
         it('uses schema #3171', () => {
           expectsql(sql.pgEnum(FooUser.getTableName(), 'mood', FooUser.rawAttributes.mood.type), {
-            postgres: 'CREATE TYPE "foo"."enum_users_mood" AS ENUM(\'happy\', \'sad\');'
+            postgres: 'DO \'BEGIN CREATE TYPE "foo"."enum_users_mood" AS ENUM(\'\'happy\'\', \'\'sad\'\'); EXCEPTION WHEN duplicate_object THEN null; END\';'
```

Comment:
```suggestion
            postgres: `DO 'BEGIN CREATE TYPE "foo"."enum_users_mood" AS ENUM(''happy'', ''sad''); EXCEPTION WHEN duplicate_object THEN null; END';`
```
This should work, although the double single quotes in the enum is strange

Reply:
There is a [problem with ESLint](https://github.com/sequelize/sequelize/blob/d3f5b5a65e297f4b6861e6a6ce335a9830b28781/.eslintrc.json#L67)

Reply:
Ah, that has been improved in `main` then. You can ignore this for this PR and only do it for the PR targeting `main`

Reply:
Ok, it's done: https://github.com/sequelize/sequelize/pull/15738